### PR TITLE
simpleai: init at 0.7.11

### PR DESCRIPTION
**Pull request by NikolaMandic**
###### Things done
- [ ] Tested using sandboxing (`nix-build --option build-use-chroot true` or [nix.useChroot](http://nixos.org/nixos/manual/options.html#opt-nix.useChroot) on NixOS)
- Built on platform(s)
  - [x ] NixOS
  - [ ] OS X
  - [ ] Linux
- [x ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -582,6 +582,22 @@ in modules // {
...
+    src = pkgs.fetchurl {
+      url= "https://pypi.python.org/packages/source/s/simpleai/simpleai-${version}.tar.gz";
+      sha256 = "03frjc5jxsz9xm24jz7qa4hcp0dicgazrxkdsa2rsnir672lwkwz";
+    };
+
+    meta = {
+      homepage = https://github.com/simpleai-team/simpleai;
+      description = "This lib implements many of the artificial intelligence algorithms described on the book 'Artificial Intelligence, a Modern Approach'";
+      maintainers = with maintainers; [ NikolaMandic ];
+    };
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: you will need to add yourself also to lib.maintainers.
https://github.com/NixOS/nixpkgs/blob/master/lib/maintainers.nix

Comment (NikolaMandic) [inline pkgs/top-level/python-packages.nix]: I hope you guys mege pybrain pull request where I did that

**Comment by NikolaMandic** [comment]
this should be ok now

**Comment by NikolaMandic** [comment]
@FRidh ping just in case it does not get seen

**Comment by FRidh** [comment]
Judging from the log
- you need to add `numpy` as `propagatedBuildInput`
- this package does not work with Python 3, and therefore it needs `disabled = isPy3k`.

**Comment by NikolaMandic** [comment]
@FRidh done

File: pkgs/top-level/python-packages.nix [hunk 2]
```diff
@@ -27338,4 +27338,25 @@ in modules // {
       license = licenses.asl20;
     };
   };
+
+  simpleai = buildPythonPackage rec {
+     version = "0.7.11";
+     name = "simpleai";
```
Comment (joachifm) [inline pkgs/top-level/python-packages.nix]: The `version` must be appended to `name`.

Comment (NikolaMandic) [inline pkgs/top-level/python-packages.nix]: @joachifm  is it good now?

**Comment by FRidh** [comment]
@NikolaMandic Good. I checked the log and while it succeeded with building I noticed that no tests were discovered. With Python packages we generally run tests after building the package. Not all packages are distributed with tests though, and `simpleai` is one of them. If that is the case it is preferred to explicitly disable checking. Could you therefore disable tests (`doCheck = false;`) and add this reason (`No tests in archive`). Thanks.

**Comment by FRidh** [comment]
@NikolaMandic and the name is good now.

**Comment by NikolaMandic** [comment]
@FRidh  done

**Pull request merged**